src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -1,31 +1,66 @@
-import React, { useCallback, useMemo } from "react";
+import React, {
+  MouseEventHandler,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
 import classNames from "classnames";
 
-import { PosterPageVenue } from "types/venues";
+import { PersonalizedPoster, VenueEvent } from "types/venues";
 
 import { WithId } from "utils/id";
 
 import { PosterCategory } from "components/atoms/PosterCategory";
 
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBookmark as solidBookmark } from "@fortawesome/free-solid-svg-icons";
+import { faBookmark as regularBookmark } from "@fortawesome/free-regular-svg-icons";
+
+import { updatePersonalizedSchedule, savePosterToProfile } from "api/profile";
+import { useUser } from "hooks/useUser";
+
+// import { UserAvatar } from "components/atoms/UserAvatar";
+// import { useProfileModalControls } from "hooks/useProfileModalControls";
+// import { useWorldUsersById } from "hooks/users";
```

Comment:
Why is this commented out here?

Reply:
This was from back when I was also integrating the user avatar into the poster preview card. Will clean up here

Reply:
done